src/Common/src/Interop/FreeBSD/Interop.Libraries.cs

```diff
@@ -5,6 +5,12 @@ internal static partial class Interop
 {
     private static partial class Libraries
     {
-        internal const string LibRt = "librt"; // POSIX Realtime Extensions library
+        /// <summary>
+        /// We aren't OS X so don't have an INODE64 suffix to entry points
+        /// </summary>
+        internal const string INODE64SUFFIX = "";
+
+        internal const string LibRt = "librt";  // POSIX Realtime Extensions library
+        internal const string libc = "libc";    // C runtime
```

Comment:
I might be misunderstanding, but I wasn't suggesting just including the Unix Interop.Libraries.cs, I suggesting included the Unix Interop.Libraries.cs, and then also including each of the platform-specific Interop.Libraries.cs as is relevant, and libc would remain in the Unix one.